docs/data-sources/equinix_fabric_network.md

```diff
@@ -10,6 +10,14 @@ description: |-
 
 Fabric V4 API compatible data resource that allow user to fetch Fabric Network for a given UUID
 
+## Example Usage
+
+```hcl
+data "equinix_fabric_network" "network_data_name" {
```

Comment:
This is perfect for how the data source is used.

Though something we're working towards is having the data source examples include outputs that provide the attribute mappings (data selections like how we do with connections getting the redundancy group) so the customers are educated on how to retrieve the different elements from the data source.

If you find it to be quick then include. If not just let me know in a comment and we'll setup a task to pickup in Jira for getting the data source docs updated all at once.

Reply:
Could we follow up on this task later? I will create a JIRA task for updating all the data sources

Reply:
Sounds good!